fork-clone-workflow.md

```diff
@@ -64,30 +65,30 @@ git-info
 ```
 * Change directory to the newly create reporitory directory (`git-info` in this example) to configure the local clone so that it
 knows where to find the remote repository from which your fork was created. Your git clone maintains a set of
-[references](git_glossary.md#references) to remote repositories. 
-git automatically creates the `origin` reference when the clone is created. The `git remote -v` command lists the known
-[references](git_glossary.md#references) and the values associated with each reference.
+[remote](git_glossary.md#references) names that point to remote repositories. 
```

Comment:
For "`[remote](git_glossary.md#references)`" "`[remotes](git_glossary.md#references)`" can we change from "#references" to "#remote"?

Reply:
Yes. I missed those two when changing 'reference(s)' to 'remote)s)'/